#! /bin/sh /usr/share/dpatch/dpatch-run
## 15-fix-fltk-include.dpatch by Raphael Bossek <bossekr@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Uses new FLTK header files.

@DPATCH@

--- ACE_wrappers/ace/FlReactor/FlReactor.cpp.orig	2004-10-25 00:08:59.932411424 +0200
+++ ACE_wrappers/ace/FlReactor/FlReactor.cpp	2004-10-25 00:09:08.272143592 +0200
@@ -6,7 +6,7 @@
 
 #if defined (ACE_HAS_FL)
 
-#include /**/ <FL/Fl.h>
+#include /**/ <FL/Fl.H>
 
 ACE_ALLOC_HOOK_DEFINE (ACE_FlReactor)
 
